cpp/tests/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ add_executable(unit-tests
         test_plane_extractor.cpp
         test_config.cpp
         test_depth_image.cpp
-        )
+        test_segmentation_image.cpp)
```

Comment:
I am quite unsure that we need this test. What does it check? It is more like an example, but we already have similar one.

Reply:
Will be deleted later

Reply:
With this "local" test I can't see whether CI successfully passes all other tests. We'll have to delete it before moving further.

Reply:
Text deleted